.ci/bump-opbeans-node.yml

```diff
@@ -40,7 +40,7 @@ conditions:
     name: Check if installed version differs
     kind: shell
     sourceid: elastic-apm-node
-    failwhen: true
+    failwhen: false
     spec:
       command: bash .ci/scripts/test-version.sh
```

Comment:
Oh... updatecli behaviour is not exactly intuitive.  That `sourceid` means "take the output from the `elastic-apm-node` "source" and append it to the "spec.command" command.

https://www.updatecli.io/docs/plugins/resource/shell/
> 💡 The source associated to this condition (the default source or through the key sourceID) is appended as the last argument to the command line unless the attribute disablesourceinput is set to true.

I guess we have to know that `sourceid` and `sourceID` are equivalent. Because if case *did* matter we'd get the same result here: a non-zero exit status from `bash .ci/scripts/test-version.sh` (no argument) or from `bash .ci/scripts/test-version.sh 4.7.2`.  The job output of the updatecli run doesn't show sufficient details to know the difference: https://github.com/elastic/opbeans-node/actions/runs/10316486283/job/28558803311